src/main/java/dev/codesupport/web/common/security/AuthorizationService.java

```diff
@@ -10,4 +11,7 @@
 
     UserDetails getUserDetailsByEmail(String email);
 
+    @PreAuthorize("hasPermission('discord', 'link_account')")
```

Comment:
Everyone should have this permission by default.

Reply:
The only restriction this adds is that you are logged in, if you aren't, it doesn't know who to link the discorded to